pkg/kubectl/cmd/cp.go

```diff
@@ -378,11 +378,8 @@ func untarAll(reader io.Reader, destFile, prefix string) error {
 }
 
 func getPrefix(file string) string {
-	if file[0] == '/' {
-		// tar strips the leading '/' if it's there, so we will too
-		return file[1:]
-	}
-	return file
+	// tar strips the leading '/' if it's there, so we will too
+	return strings.TrimLeft(file, "/")
```

Comment:
isn't there a "clean" func in `os/path` ?

Reply:
Yes, and it is used after calling this function (https://github.com/tomerf/kubernetes/blob/52f02c42e6a84a551b0f7eade38be62dd6c37cf2/pkg/kubectl/cmd/cp.go#L236).
But it only make the path canonical, it doesn't remove prefix slash.

Reply:
@sttts Is that okay?